lib/index.js

```diff
@@ -13,10 +13,10 @@ const TaskState = require('./models/TaskState');
 const bulletStyleSetup = require('./bulletStyleSetup');
 const quit = require('./quit');
 
-const tasksFile    = path.join(os.homedir(), 'lol-tasks.json');
-const settingsFile = path.join(os.homedir(), 'lol-settings.json');
-
 const term = terminal;
+const yaml = require('js-yaml');
+const tasksFile    = path.join(os.homedir(), '.lol_task', 'tasks.json');
+const settingsFile = path.join(os.homedir(), '.lol_task', 'config.yaml');
```

Comment:
Keeping the original ordering.
```suggestion
const tasksFile    = path.join(os.homedir(), '.lol_task', 'tasks.json');
const settingsFile = path.join(os.homedir(), '.lol_task', 'config.yaml');

const term = terminal;
```